### ISSUE TITLE
Asset Management

### ISSUE THREAD

**Issue by CodeFluent** [SIFsatGSU/3DAtlanta#2]
Many articles and videos I've read and watched specify that filesize doesn't really matter in terms of Unity for assets (pngs, jpegs, bitmap works regardless).

It'd be great to have all our geometry in .blend files we can readily edit inside Blender and see the changes in Unity instantly, but I wonder if doing it the .blend way and the adding materials in the Materials folder, images in another, or whatever is better.

Basically the question is whether or not down the road hosting the assets as .blend files will come back to bite us.
